lib/model/bloc/startup/StartupViewModelProvider.dart

```diff
@@ -65,21 +68,34 @@ class StartupViewModelProvider implements ViewModelProvider<StartupViewModel> {
   }
 
   ChatPageViewModel _chatPageViewModel() {
-    return ChatPageViewModel(
-        _LocalisedAssets.onboardingFlow(), (data) {
-          final ChatResponseViewModel name = castOrNull<ChatResponseViewModel>(data["Name"]);
-          if (name != null){
-            _accountRepository.create("Test").then((account) {
-              final newProfile = ProfileEntity("test", name.value, MockImageEntity().build().urlProvider,plotInfo);
-              account.profileRepository.add(newProfile).then((profile){
-                account.profileRepository.switchTo(profile);
-                _refresh();
-              }) ;
-            });
-          }
-        },(data) {
-            //TODO: error case, should display a popup.
+    //TODO: Remove the Mock ID´s once implemented
+    final transformer = ChatResponseToPlotInfoTransformer();
+    return ChatPageViewModel(_LocalisedAssets.onboardingFlow(), (data) {
+      final ChatResponseViewModel name =
+          castOrNull<ChatResponseViewModel>(data[_Strings.nameField]);
+      if (name != null) {
+        _accountRepository
+            .create(
+          mockPlainText.identifier(),
+          mockPlainText.identifier(),
+        )
+            .then((account) {
+          final plotInfo = transformer.transform(from: data);
+          final newProfile = ProfileEntity(
+            mockPlainText.identifier(),
+            name.value,
+            MockImageEntity().build().urlProvider,
+            plotInfo,
+          );
+          account.profileRepository.add(newProfile).then((profile) {
+            account.profileRepository.switchTo(profile);
+            _refresh();
+          });
         });
+      }
+    }, (data) {
+      //TODO: error case, should display a popup.
```

Comment:
This would be done in another PR?

Reply:
yes, not sure how to deal with this ATM. not sure what an error would be. If the user quit, then they would just land back at the screen they started, so we don't need to do anything.